projects/libfido2/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: https://github.com/Yubico/libfido2
+primary_contact: "g.kihlman@yubico.com"
+auto_ccs:
+        - "pedro@ambientworks.net"
```

Comment:
Stupid nit: I think we usually use less spaces in our yaml files: see https://github.com/google/oss-fuzz/blob/master/projects/bignum-fuzzer/project.yaml for example.